### PR TITLE
Akshay docs

### PR DESCRIPTION
**Pull request by GanadiniAkshay**

### CODE REVIEW

File: docs/bot-analytics/basic-analysis.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Basic Analysis
+---
+
+## Introduction
+The Analytics tool built by haptik enables you to get a comprehensive idea of how your chatbot is performing, how users are using it, where it's failing, what the user think of it etc. 
+
```
Comment (nikunj1705) [inline docs/bot-analytics/basic-analysis.md]: Break this line down. Also, ".. what the user **thinks** of it.."

File: docs/bot-analytics/basic-analysis.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Through this document we will explore the different types of metrics available for tracking and analysing inside the analytics tool.
+
+The Analytics tool consists of multiple section with each section containing metrics and data about a different aspect of your chatbot. These different sections are:
+
```
Comment (nikunj1705) [inline docs/bot-analytics/basic-analysis.md]: Multiple sections*

File: docs/bot-analytics/basic-analysis.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+The Overview section provides information about:
+* Conversations
+* Users
+* User Messages
+* Task Completion
+* Bot Automation
+* User Rating
+
+You can later use a different section to get more detailed information on any of these sub sections.
+
```
Comment (nikunj1705) [inline docs/bot-analytics/basic-analysis.md]: Hyperlink each section to their detail pages?

File: docs/bot-analytics/basic-analysis.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+## ***User Data***
+The next tab in the overview section is the Users tab. This tab shows the number of users for the filtered channel or bot in the given duration period.
+
+The graph for this data is an upcoming feature and will be documented here once it is available.
+
+![Overview Users](assets/basic-overview-users.png)
+
+## ***User Messages***
+The next tab in the Overview section is the User Messages tab. This tab measures what tasks where most initiated by user. It shows a list of tasks in the chatbot along with the number of users who initiated the task.
+
```
Comment (nikunj1705) [inline docs/bot-analytics/basic-analysis.md]: ..**were** most initiated by user..

File: docs/bot-analytics/basic-analysis.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+## ***Task Completion***
+The next tab is task completion. While the previous tab shows the task activation rate, this tab shows the task completion rate. Task completion rate is the percentage of users who completed a task after activating it.
+
+On the top you can see the total task completion rate in percentage format. Below it the data is split from a task to task basis.
+
+![Overview Task Completion](assets/basic-overview-task-completion.png)
+
+>Tip: Make improvements to the task flow and analyze the change of task completion rate to see how the improvement is performing.
+
```
Comment (nikunj1705) [inline docs/bot-analytics/basic-analysis.md]: ... the change **in** task completion rate...

Comment (nikunj1705) [reply docs/bot-analytics/basic-analysis.md]: rephrase this sentences. The sentences should be smaller. Easier to understand with min. conjunctions.

File: docs/bot-analytics/basic-analysis.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+The graph for this tab is an upcoming feature and will be documented here once it is ready.
+
+## ***User Rating***
+The final tab is User Rating. This tab tells you how users rated the different tasks in the chatbot. The rating is optional and is given on a scale of 1-5 stars.
+
+On the top of the tab you can see the average rating given across all tasks and below it you have the data split from a task to task basis.
+
+![Overview User Rating](assets/basic-overview-user-rating.png)
+
+> Tip: You can try to improve the task flow and see how the user rating changes for the task accordingly to measure your improvements.
```
Comment (nikunj1705) [inline docs/bot-analytics/basic-analysis.md]: Breakdown this tip into 2 sentences.

File: docs/bot-analytics/basic-analysis.md (hunk 3 above)
Comment (GanadiniAkshay) [reply docs/bot-analytics/basic-analysis.md]: They don't exist as of now...maybe we will take it up in cleanup

File: docs/bot-analytics/basic-analysis.md (hunk 2 above)
Comment (nikunj1705) [reply docs/bot-analytics/basic-analysis.md]: The Analytics tool consists of multiple sections*

File: docs/bot-analytics/basic-analysis.md (hunk 5 above)
Comment (nikunj1705) [reply docs/bot-analytics/basic-analysis.md]: This metric can guide you **to understand** how those changes are performing.

**Pull request merged**